README.md

```diff
@@ -36,6 +36,22 @@ $ ccm start
 # Wait a lot...
 ```
 
+### Create a cluster of nodes with a specific build-id
+
+Let's say you want to create a cluster of `Scylla 2023.1.0~rc8-20230731.b6f7c5a6910c`, with build id `f6e718548e76ccf3564ed2387b6582ba8d37793c`.
```

Comment:
It sounds like you are expected to know exactly which build id you are looking for, but in fact the first step below tells you how to find that build id. Maybe you should start saying you want "Scylla 2023.1.0" and then below you'll be told to search for that in backtrace.scylladb.com.

Reply:
I have the `build-id` and I want to find a download for the package with this build-id, so I use https://backtrace.scylladb.com to get the download link based on the `build-id` alone.

I modified this line to focus on the `build-id` more.